### PR TITLE
Changing error message when setting pricing rules

### PR DESCRIPTION
**Pull request by HemaHG**
**What this PR does / why we need it**:

changing the error message when setting pricing rules

**Which issue(s) this PR fixes** 

https://issues.redhat.com/browse/THREESCALE-5565

**Verification steps** 
Screenshot from 2020-07-21 14-40-48

### CODE REVIEW

File: app/models/pricing_rule.rb [hunk 1]
```diff
@@ -88,7 +88,7 @@ def range_overlap
     end
 
     if overlap #rules.count > 0
-      errors.add(:min, "'From' value cannot be less than 'To' values of current rules.")
+      errors.add(:min, "'From' value must be greater than 'To' values of current rules.")
```
Comment (thomasmaas) [inline app/models/pricing_rule.rb:91]:
```suggestion
      errors.add(:min, "The 'From' value must be greater than the 'To' values of other existing rules.")
```

Comment (Martouta) [reply app/models/pricing_rule.rb:91]: Plus, this is a good moment to use i18n and put this in locales 😄

File: app/models/pricing_rule.rb [hunk 2]
```diff
@@ -88,11 +88,11 @@ def range_overlap
     end
 
     if overlap #rules.count > 0
-      errors.add(:min, "'From' value cannot be less than 'To' values of current rules.")
+      errors.add(:min, :overlap_message)
     end
 
     if max && max < min
-      errors.add(:max, "'To' value cannot be less than your 'From' value.")
+      errors.add(:max, :to_value_cannot_be_less_that_from_value)
```
Comment (Martouta) [inline app/models/pricing_rule.rb:95]:
```suggestion
      errors.add(:max, :min_max_mismatch)
```

File: config/locales/en.yml [hunk 3]
```diff
@@ -843,6 +843,13 @@ en:
             version:
               mismatch: "differs from the schema version"
               builtin: "cannot be 'builtin'"
+        pricing_rule:
+          attributes:
+            min:
+              overlap_message: "The 'From' value must be greater than the 'To' values of other existing rules"
+            max:
+              to_value_cannot_be_less_that_from_value: "'To' value cannot be less than your 'From' value."
```
Comment (Martouta) [inline config/locales/en.yml:851]:
```suggestion
              min_max_mismatch: "'To' value cannot be less than your 'From' value."
```

File: app/models/pricing_rule.rb [hunk 4]
```diff
@@ -88,11 +88,11 @@ def range_overlap
     end
 
     if overlap #rules.count > 0
-      errors.add(:min, "'From' value cannot be less than 'To' values of current rules.")
+      errors.add(:min, :overlap_message)
```
Comment (Martouta) [inline app/models/pricing_rule.rb:91]:
```suggestion
      errors.add(:min, :overlap)
```

File: config/locales/en.yml [hunk 5]
```diff
@@ -843,6 +843,13 @@ en:
             version:
               mismatch: "differs from the schema version"
               builtin: "cannot be 'builtin'"
+        pricing_rule:
+          attributes:
+            min:
+              overlap_message: "The 'From' value must be greater than the 'To' values of other existing rules"
```
Comment (Martouta) [inline config/locales/en.yml:849]:
```suggestion
              overlap: "The 'From' value must be greater than the 'To' values of other existing rules"
```

**Pull request merged**